Mod/sources/address-bar.css

```diff
@@ -208,8 +221,8 @@ path[d^="M3 4h10v1H3zM3"] {
 }
 .SiteInfoButton.secure,
 .SiteInfoButton.certified {
-  fill: var(--colorAccentFg);
-  color: var(--colorAccentFg);
+  fill: var(--GXbuttonFill);
+  color: var(--GXbuttonFill);
```

Comment:
This change is more of a personal preference. I believe @beyxnd wanted it to match the URL address, but since the color of all the address bar icons are deviating from Opera GX, there isn't really a right answer.

Reply:
@BPZ3, do you have any thoughts on changing the padlock icon to match the other address bar buttons' fill?